DevCycle/Models/PlatformDetails.swift

```diff
@@ -9,6 +9,8 @@ import Foundation
 
 #if os(iOS)
 import UIKit
+#elseif os(OSX)
+import IOKit
```

Comment:
Would we need to add this as a framework dependency to the SDK?

Reply:
going to merge and try it out.

Reply:
my podfile was not changed so I dont think so